src/time.jl

```diff
@@ -23,13 +23,9 @@ function every(dt)
     n
 end
 
-function weakrefdo(ref, yes, no=()->nothing)
```

Comment:
The basic idea of `fps` is that it will fire at most `rate` times a second. If a signal depending on this fps signal takes longer than `1/rate` seconds to update (for example drawing a complex Compose image), it will try to give you the best possible frame rate. The implementation schedules an update `1/rate` seconds from the previous update (during the previous update) to accomplish this. The fps signal itself contains the time deltas between the previous update and the current one...